keps/sig-apps/3998-job-success-completion-policy/kep.yaml

```diff
@@ -5,7 +5,7 @@ authors:
 owning-sig: sig-apps
 participating-sigs:
   - sig-apps
-status: implementable
+status: implemented
```

Comment:
AIUI we use `implemented` once the feature is generally available.

Reply:
But this is still alpha

Reply:
Thank you for checking this!
I didn't know that. I'm updating it!

Reply:
I dropped this commit.